language-design.md

```diff
@@ -17,7 +17,20 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+***Quote 1***
+>"I now think that I, as a language designer who helps out with the design of the Java programming language, need to ask not 'Should the Java programming language grow?’ but ‘How should the Java programming language grow?'" [Steele, 1998, 7]
 
+When boiled down to its very essence, this quote encapsulates the overall philosophy of this paper as well as Steele’s advice to all language designers. Specifically, this quote talks of the importance of language growth and evolution. A programming language does not exist to be a rigid set of unchanging rules. Rather, it exists as a tool for enabling powerful communication between humans and machines. Therefore, if there is a way to better achieve this communication, the language needs to adapt to the needs of its users and enable maximal ease and expressiveness. For these reasons, language evolution is not optional, but a necessity that motivates Steele’s claim that the “…main goal in designing a language should be to plan for growth” [Steele, 1998, 7]. Although Steele goes on to explain ideal methodologies for growing a language, citing the Linux development model as an exemplary case, the principal goal comes back to prioritizing the needs of the users. It is for this reason that Steele advocates creating languages as a more accessible platform that enables extensibility and natural evolution through things like libraries and extendable syntax.
```

Comment:
I agree Steele is saying this and that this is a good microcosm of the paper as a whole, but I can't tell from this to what extent you agree with him. While I think I tend to agree with him that extensibility and user-focused design are admirable goals, I'm still not certain if they're the only goals that need to be put at the forefront, particularly extensibility. I'm curious whether or not you agree with his line of thought here.